### DISCUSSION TITLE
Keycloak get all users returns 403 and message "unknown_error"

### DISCUSSION THREAD

**Discussion by SilvandroOliveira** [keycloak/keycloak#21235] [Q&A]
I'm using version 21.1.1 of Keycloak and I'm trying to list the users from the endpoint https://{myDomain}/admin/realms/{myRealm}/users, I pass the Authorization header with the bearer token obtained, but I always get I see a 403 error - "unknown_error" in the tests I do through Insomnia, does anyone have any idea what it could be?

**Comment by Leo0426** [comment]
Make sure you had a permission to get user list. Your token should have role of manage-users.
